projects/kit/components/input-date/native-date/native-date.component.ts

```diff
@@ -0,0 +1,37 @@
+import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
+import {DATE_FILLER_LENGTH, TUI_DATE_FORMAT, TuiDateMode, TuiDay} from '@taiga-ui/cdk';
+import {TUI_TEXTFIELD_HOST, TuiTextfieldHost} from '@taiga-ui/core';
+
+@Component({
+    selector: 'input[tuiDate]',
+    template: '',
+    host: {
+        type: 'date',
+        '[tabIndex]': 'host.focusable ? 0 : -1',
```

Comment:
I don't think it would matter, but it probably should always be -1 since we have original input too.